config/csm/test_rest_capacity.yaml

```diff
@@ -84,4 +102,4 @@ test_39923:
   obj_size: 12288
   num_clients: 2
   num_sample: 5
-  obj_name_pref: "load_gen"
+  obj_name_pref: "load_gen"
```

Comment:
please add empty line at the end of file.

Reply:
done